courses/machine_learning/deepdive2/recommendation_systems/solutions/content_based_by_hand.ipynb

```diff
@@ -0,0 +1,525 @@
+{
```

Comment:
<blockquote>Initialize our users, movie ratings and features</blockquote><p>Add comma in title after "movie ratings". </p><p> </p><blockquote>Each row represents one of the six movies, the columns represent the five categories.</blockquote><p>Replace comma with semi-colon, and, or while.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/691/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Oxford comma? Eh, I'm not too picky :)</p>

Reply:
Ack.